### ISSUE TITLE
Bugged Triggers Druid Forms SoM

### ISSUE THREAD

**Issue by silverselo** [WeakAuras/WeakAuras2#3429]
### Description

![image](https://user-images.githubusercontent.com/43469073/149616449-72915b5d-92e3-4e95-a7d3-cd820ae0569b.png)
![image](https://user-images.githubusercontent.com/43469073/149616596-e4becb2e-0709-4473-b9df-26730d106787.png)

This is the trigger of the Weakaura, when I export it to demonstrate my problem, all druid forms are ticked on , as you can see here https://wago.io/Gd0FR_qpN. 

Now this problem happens only with the Cat form of the druid, it seems like the Form/stance IDs aren't correct . There seems to be a problem with the IDs in SoM. If I switch to Bear Form the Healing Touch Spell isn't displayed.

### WeakAuras Version

WeakAura 3.7.5

### World of Warcraft Flavor

Classic

### Tested with only WeakAuras

- [X] Yes
- [ ] No

### Lua Error

No LuA Error

### Reproduction Steps

make a WA like this and export it, the triggers will be changed.
![image](https://user-images.githubusercontent.com/43469073/149616449-72915b5d-92e3-4e95-a7d3-cd820ae0569b.png) 
![image](https://user-images.githubusercontent.com/43469073/149616604-9dd99cb5-5fdb-47ba-a734-c3dad1c98de3.png)

https://wago.io/Gd0FR_qpN (here are all triggers on)

**Comment by mrbuds** [comment]
I tested both in cat form and travel form (your text say "cat" but the picture show "travel") and i could not reproduce your problem.
I did my tests on TBC because i don't have any character on SoM, would you be able to try on TBC?

**Comment by silverselo** [comment]
Yes, will try it in TBC as soon as I can.

**Comment by silverselo** [comment]
![image](https://user-images.githubusercontent.com/43469073/149625196-a67eee23-f802-4def-add1-2b44db8e8a9c.png)
now in Cat form
![image](https://user-images.githubusercontent.com/43469073/149625234-d2550402-3729-41f0-92d8-4436b2103203.png)
and it is hidden as it should be
![image](https://user-images.githubusercontent.com/43469073/149625247-330b3659-486f-49e0-b8b3-4c884532f865.png)
![image](https://user-images.githubusercontent.com/43469073/149625266-828cf1a8-b88b-4b04-8a60-4d85e3bd2644.png)

is it bugging because I don't have aquatic form in SoM yet?

**Comment by mrbuds** [comment]
I don't understand your answer, is it working on tbc or not?

**Comment by silverselo** [comment]
sorry, it is working in TBC, I haven't ticked Cat Form , and it hides it.
it isn't doing that in SoM though

**Comment by InfusOnWoW** [comment]
So no one on the development team has a druid on any SOM server, and since forms need to be unlocked while leveling, we can't easily test that. Thus we need your assistance in figuring out what exactly goes wrong. For that we need some debug output from the apis we use.

We are using the following apis:
To determine the current form, we call GetShapeshiftForm().
So can you run "/dump GetShapeshiftForm()" while in each from. A screenshot is good enough.

**Comment by silverselo** [comment]
(Dire)Bear
![WoWScrnShot_012122_151220](https://user-images.githubusercontent.com/43469073/150541604-48ae914c-15e2-4909-ad45-5f2b5d5a7a65.jpg)


Cat
![WoWScrnShot_012122_151129](https://user-images.githubusercontent.com/43469073/150541591-fd576037-5670-437e-8758-4ccd52156b7a.jpg)


Travel
![WoWScrnShot_012122_151254](https://user-images.githubusercontent.com/43469073/150541612-083f7454-864f-49b0-9a94-df5c4daf9769.jpg)

Aquatic Form
![WoWScrnShot_012122_160236](https://user-images.githubusercontent.com/43469073/150549554-398220cc-1acc-4d3d-abd8-1d7693fcd250.jpg)


Also I remeber when I switched from TBC Druid to SoM Druid , I had to change some of my macros because the Form IDs where different. Hope this helps

**Comment by silverselo** [comment]
It seems to working, after getting my aquatic form.

**Comment by InfusOnWoW** [comment]
Okay, that it fixes itself with Aquatic form might mean that the bug is also present in bcc, and that we have tested something different.

Additionally that sounds like either a bug in how World of Warcraft works, or in how we use the api. I'm more leaning towards the former because if I unde

But we would need a druid with shape shift forms and without aquatic form to figure this out and test any fixes, and we don't have that. Nor can we ask you anymore, since it you apparently have the form now. So for now, there's nothing we can do.

**Comment by mrbuds** [comment]
In the screenshots GetShapeshiftForm return 2 for both cat form and aquatic form, i guess you learned aquatic form in between.
Anyway that doesn't explain the original issue you reported, which was as i understood: check 2 choices in stance trigger => export => all choices are checked on import

**Comment by silverselo** [comment]
The first 3 screenshots were before I got the Aquatic Form, I got the Aquatic Form just yesterday for this occasion.
![WoWScrnShot_012222_071036](https://user-images.githubusercontent.com/43469073/150627231-e5f226df-2798-46c0-93bc-d09b9a6bb283.jpg)
Here is the Cat Form now after I got the Aquatic Form. It went from 2 ----> 3

**Comment by silverselo** [comment]
Travel Form 3 ---> 4 
![WoWScrnShot_012222_071316](https://user-images.githubusercontent.com/43469073/150627298-a66d6bea-aa25-4a50-8574-88c68ffb0ae3.jpg)
